Source/JavaScriptCore/runtime/RegExpSubstringGlobalAtomCache.cpp

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright (C) 2024 Apple Inc. All rights reserved.
```

Comment:
Let's add V8 copyright too.

Reply:
Added the V8 commit to this commit message.